server/.env.example

```diff
@@ -136,6 +136,10 @@ PA_PASSWORD_SECONDARY_TEST=ABpassword
 DOMAIN=domain.brave.coop
 DOMAIN_TEST=domain.brave.coop
 
+# API Key for Particle Webhook to make API calls to Heartbeat
```

Comment:
This comment specifies heartbeat, but it's also used in the other calls. Could you please change it either so that it includes those explicitly or make it more general by removing the Heartbeat part?

Reply:
thanks for catching that! really appreciate it : )